backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/init/CassandraConfigurationReadingTest.java

```diff
@@ -43,7 +43,7 @@ void provideCassandraConfigurationShouldReturnRightConfigurationFile() throws Co
         FileBasedConfigurationBuilder<FileBasedConfiguration> builder = new FileBasedConfigurationBuilder<FileBasedConfiguration>(PropertiesConfiguration.class)
             .configure(new Parameters()
                 .fileBased()
-                .setURL(ClassLoader.getSystemResource("cassandra.properties")));
+                .setURL(ClassLoader.getSystemResource("configuration/cassandra.properties")));
```

Comment:
I already fall on that, it's a good idea to move it.
Could we use a path name that make it easier to find the related test, like `configuration-reader-test`